workflows/pipe-common/scripts/autoscale_sge.py

```diff
@@ -249,6 +249,9 @@ def get_jobs(self):
                 current_host = None
         return jobs.values()
 
+    def filter_running_job(self, jobs):
+        return [job for job in jobs if job.state == GridEngineJobState.RUNNING]
+
```

Comment:
I believe it's okay to inline this method since it really depends on the call site. Otherwise we can introduce `get_running_jobs` method which can be optimized later on to retrieve only running jobs from the qstat. But for now I would just inline the method.

Reply:
Done